server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1436,6 +1441,86 @@ public void run() {
     log.info("exiting");
   }
 
+  /**
+   * Allows property configuration to block master start-up waiting for a minimum number of tservers
+   * to register in zookeeper. It also accepts a maximum time to wait - if the time expires, the
+   * start-up will continue with any tservers available. The following properties are used to
```

Comment:
Should add `if the time expires, the start-up will continue with any tservers available` to the property documentation.